libraries/aws/provisioning/src/aws_iot_provisioning_serializer.c

```diff
@@ -44,6 +44,22 @@
     #error "AWS_IOT_PROVISIONING_ENABLE_ASSERTS must be 0 or 1."
 #endif
 
+/*------------------------------------------------------------------*/
+
+/**
+ * @brief Represents the number of map entries in the request payload of the
+ * MQTT CreateCertificateFromCsr service API.
```

Comment:
The name for doesn't seem like it's for the CreateCertificateFromCsr API. Was the description copied over from below?

Reply:
Yes, copy-pasta in the coding spree.

Reply:
Fixed.